tests/regression/test_multiplex.py

```diff
@@ -223,14 +226,61 @@ def test_proteinase_k_dose(
             )
 
 
-@pytest.mark.parametrize(
-    "parameters",
-    (
-        ["min_completeness=0.5", "cluster_method=cos_angle"],
-        ["min_completeness=0.5", "cluster_method=correlation"],
-    ),
-)
-def test_proteinase_k_min_completeness(parameters, proteinase_k, run_in_tmp_path):
+def test_proteinase_k_coordinate_clusters(proteinase_k, run_in_tmp_path):
+    expts, refls = proteinase_k
+    parameters = [
+        "clustering.output_clusters=True",
+        "clustering.method=coordinate",
+        "symmetry.space_group=P422",
+    ]
+    command_line_args = parameters + expts + refls
```

Comment:
I think this is unstable from memory, let's use expts[:-1] and refls[:-1] and then will need to reduce the assert in L243 to 7.

Reply:
Have put this in - lets hope it fixes the tests :)